pkgs/bytecodealliance/wasm-pkg-tools/registry.yaml

```diff
@@ -0,0 +1,5 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/aquaproj/aqua/main/json-schema/registry.json
+packages:
+  - type: github_release
+    repo_owner: bytecodealliance
+    repo_name: wasm-pkg-tools
```

Comment:
Seems like the code wasn't scaffolded properly.
I re-scaffolded. e486c32bb9db9359a9a5f60ea326af438fe9eec5

```sh
cmdx s -B -cmd wkg bytecodealliance/wasm-pkg-tools
```